R/utilities.R

```diff
@@ -1320,14 +1320,10 @@ collate_results = function(sample_table,
   if(write_to_file){
     from_cache = FALSE #override default automatically for nonsense combination of options
   }
-  output_file = config::get("results_merged")$collated
```

Comment:
This is a partial fix but how will this work when use_cache = TRUE?

Reply:
I know you said it's only fixing one of the two modes but this code change is guaranteed to not allow the other mode to ever work due to the scope of the variables.

Reply:
So I guess both options (from_cache = TRUE/FALSE) would work if they have access to the same output_files? i.e moving the output_file code outside the from_cache if statement. Allowing from_cache = FALSE to find the file if write_to_file = TRUE. Is this correct?

```
 output_file = config::get("results_merged")$collated
 output_base = config::get("project_base")
 output_file = paste0(output_base, output_file)
 output_file = glue::glue(output_file)
```